pokedex/data/csv/items.csv

```diff
@@ -953,3 +953,135 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
 1003,roto-pp-restore,23,0,,
 1004,roto-boost,23,0,,
 1005,roto-catch,23,0,,
+1006,autograph,,0,,
+1007,pokemon-box,,0,,
+1008,medicine-pocket,,0,,
+1009,candy-jar,,0,,
+1010,power-up-pocket,,0,,
+1011,clothing-trunk,,0,,
+1012,catching-pocket,,0,,
+1013,battle-pocket,,0,,
+1014,silver-razz-berry,,1000,10,
+1015,golden-razz-berry,,5000,10,
+1016,silver-nanab-berry,,1000,10,
+1017,golden-nanab-berry,,5000,10,
+1018,silver-pinap-berry,,1000,10,
+1019,golden-pinap-berry,,5000,10,
+1020,secret-key,,0,,
+1021,ss-ticket,,0,,
```

Comment:
I added a `--letsgo` suffix, mainly not to interfere with item_game_indices.csv, since the game data seems to have both the new and the original items